lib/game.rb

```diff
@@ -18,4 +18,8 @@ def initialize(row)
     @away_goals = row[:away_goals].to_i
     @home_goals = row[:home_goals].to_i
   end
+
+  def score_count
```

Comment:
Could we call this total_score to keep it consistent with the highest and lowest total score methods?